examples/getstarted/src/plugins/mycustomfields/admin/src/components/ColorPicker/ColorPickerInput/index.js

```diff
@@ -18,19 +19,22 @@ const ColorPickerInput = ({
   const { formatMessage } = useIntl();
 
   return (
-    <Stack spacing={1}>
-      <Field
-        name={name}
-        id={name}
-        error={error && formatMessage(error)}
-        hint={description && formatMessage(description)}
-      >
+    <Field
+      name={name}
+      id={name}
+      error={error && formatMessage(error)}
+      hint={description && formatMessage(description)}
+    >
+      <Stack spacing={1}>
         <FieldLabel required={required}>{formatMessage(intlLabel)}</FieldLabel>
+        <Typography variant="pi" as="p">
+          Format: {attribute.options.format}
```

Comment:
Could you add a translation for this too?

Reply:
Or I guess maybe it's not necessary since we are going to re-develop all this anyway...

Reply:
Sure